### ISSUE TITLE
Junit4 @Category not working

### ISSUE THREAD

**Issue by GoGoris** [concordion/concordion#153]
Hi, 

It seems like the @Category annotation on my Concordion fixture does nothing, is this correct? I already tried manually adding these annotations to my FakeFrameworkMethod, but without success so far.

**Comment by nigelcharman** [comment]
Yes, it's not something that we've added support for, and it requires a custom JUnit runner. With JUnit 5 changing the way that extensions are added, I'd be loathe to invest much time in this at the moment.

There's a long-running enhancement request (#10) for tagging specifications that I'd like to see us address this year.

In the meantime, a potential workaround is to create custom suites of tests, either using the concordion:run command, or JUnit suites.

**Comment by GoGoris** [comment]
OK maybe something for after Junit 5 than.

**Comment by niegus** [comment]
Will this be on the agenda in the near future? It will very useful for our current project and will save us a lot of time.
